package/libpiksi/libpiksi/include/libpiksi/util.h

```diff
@@ -77,6 +77,8 @@ bool device_has_gps_time(void);
  */
 void set_device_has_gps_time(bool has_time);
 
+bool file_read_value(char *file_path);
```

Comment:
Looks like this isn't needed after all in this PR but I think it's a good function anyway so I propose to merge it anyway.